### PR TITLE
Proof reading Upgrade from Open Distro to OpenSearch.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@amazon.com>

### Description

Blog post is great! Some proof-reading.
 
By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by elfisher — APPROVED** [review]
these LGTM. Good catches @dblock

File: _posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown [hunk 1]
```diff
@@ -214,40 +214,42 @@ Verify all the cluster health, indices, settings etc are seamlessly migrated and
...
+OpenSearch can be upgraded using a Rolling Upgrade for most minor and major versions, thus upgrading going forward will not usually interrupt your service.
 
-For more information on upgrading please see our [Upgrading FAQs](https://opensearch.org/faq/#c3) and if you have questions about backward compatibility you can check dB’s post on [Backwards Compatibility FAQ](https://opensearch.org/blog/technical-posts/2021/06/opensearch-backwards-compatibility-faq/). If you’re looking for a great “getting started” guide I’d recommend Gedalyah’s [Install and Configure OpenSearch guide](https://logz.io/blog/opensearch-tutorial-installation-configuration/).
+For more information on upgrading please see our [Upgrading FAQs](https://opensearch.org/faq/#c3). If you have questions about backward compatibility, check out [this blog post](https://opensearch.org/blog/technical-posts/2021/06/opensearch-backwards-compatibility-faq/). If you’re looking for a great "Getting Started" guide, I highly recommend Gedalyah’s [Install and Configure OpenSearch](https://logz.io/blog/opensearch-tutorial-installation-configuration/).
```
Comment (stockholmux) [inline _posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown:251]: We should avoid links like 'this blog post' as it bad for SEO and accessibility (screen readers). The link text should describe subject matter of the link.

**Review by stockholmux — CHANGES_REQUESTED** [review]
Small change in link text.

File: _posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown (hunk 1 above)
Comment (dblock) [reply _posts/2021-07-08-how-to-upgrade-from-opendistro-to-opensearch.markdown]: Fixed.

**Review by elfisher — APPROVED** [review]
lgtm!

**Comment by elfisher** [comment]
@stockholmux it looks like your requested changes were addressed. Pls give approval if this looks good to you now.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**